unit-4-dom-labs/menu-lab/index.css

```
@@ -0,0 +1,53 @@
+header {
+	height: 20vh;
```

```suggestion
header {
```

This is a very tall header for a small button icon